CHANGELOG.md

```diff
@@ -1,5 +1,12 @@
 # CHANGELOG
 
+## 3.0.0 - 2019-10-14
```

Comment:
I would vouch for "not released" or something like this to avoid misleading information if anyone look at this file on master.